README.md

```diff
@@ -1,2 +1,38 @@
 # told_ya
+
 Project for Starkhack 2024
+
+## Test / deploy
+
+### Requirements
+
+Requirements:
+
+- scarb 2.6.3 (e6f921dfd 2024-03-13)
+- cairo: 2.6.3 ([https://crates.io/crates/cairo-lang-compiler/](https://crates.io/crates/cairo-lang-compiler/))
+- sierra: 1.5.0
+- make (gcc)
+- starkli: 0.2.9
+- katana: 0.7.2 (dev dependency)
+
+### Setup
+
+Install the requirements:
+
+```bash
+make setup
```

Comment:
Should be setup-unix to be compliant with Makefile

Reply:
Right indeed, wrote that one way too fast ahah 💀